src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -147,10 +147,10 @@ SELECT c.concept_name, c.concept_id, c.concept_code, COUNT(DISTINCT co.person_id
   @ParameterizedTest
   @ArgumentsSource(SqlRenderContextProvider.class)
   void testCreateSearchConceptClause(SqlRenderContext context) {
-    TableVariable conceptTableVariable =
-        TableVariable.forPrimary(TablePointer.fromTableName(Concept.TABLE_NAME));
+    SourceVariable conceptSourceVariable =
```

Comment:
To me, this shows why it's not necessarily good to name variables after their types. In this code's context, is the type more important than the thing it represents? Maybe now is a good time to rename it to `conceptTable` so it refers to its content instead of its type.
```suggestion
    SourceVariable conceptTable =
```